docs/course/index.md

```diff
@@ -493,3 +485,35 @@ While every course should use the different roles in a way that best suits the c
 | TAs involved in grading              |         None          | Student data editor |
 | Other TAs                            |         None          | Student data viewer |
 | Instructors from other classes       | Course content viewer |        None         |
+
+## Course-wide options
```

Comment:
I needed to remove the link to the other page, but I wasn't comfortable having zero information about what the new vs. legacy renderers were, so I included a pared-down version here that describes the primary thing to consider when moving to the new renderer: no support for self-closing element tags. I also moved this whole thing to the bottom of the doc since it's going to be irrelevant to users 99% of the time.

Reply:
Since the check for the feature flag comes before the check for this option, is it really relevant?

https://github.com/PrairieLearn/PrairieLearn/blob/4099a2df4f77c1da59e6580ab55c0312de6d0e65/apps/prairielearn/src/question-servers/freeform.js#L1886-L1890

I assume any course where this option is relevant (i.e., any course that has explicitly been set to use the previous renderer) is either already aware of this option or is being handled directly. So for any new course using the default options (i.e., the experimental renderer), this setting is not going to change anything.